docs/source-app/glossary/command_lines/command_lines.rst

```diff
@@ -0,0 +1,76 @@
+:orphan:
+
+############################
+Command-line Interface (CLI)
+############################
+
+**Audience:** Users looking to create a command line interface (CLI) for their application.
+
+----
+
+**************
+What is a CLI?
+**************
+
+A CLI is a text-based user interface (UI) used to run programs, manage computer files and interact with the computer. CLIs are also called command-line user interfaces, console user interfaces and character user interfaces.
```

Comment:
We need a citation or write it in our own words.

Reply:
Done !